int16 angle_speed 0
int16 height_speed 0
int16 distance_speed 0
float32 gripper_yaw 90
float32 gripper_pitch 90
float32 claw_angle 40
